technical-vision.md

```diff
@@ -7,3 +7,66 @@ We envision a future where participants in the open source ecosystem use and sha
 - Community members can provide information and notifications about product defects, mitigations, quality, and supportability and have this information rapidly flow forward across the ecosystem system to all users, and users can rapidly update their software or implement mitigations as appropriate.
 
 The Open Source Security Foundation (OpenSSF) provides tools, services, training, infrastructure, and resources to achieve this vision.
+
+---
+
+# OpenSSF Technical Vision (v2 proposed)
+
+> _Note: the bullet points that are included below in block quotes would not be included in the final version of the vision, but are listed for reference/discussion._
+
+An aspirational view of the future we seek to create in 2-5 years:
+
+### OpenSSF is a well-respected strategic partner to both large OSS foundations and individual OSS projects
+- Clearly positioned within broader ecosystem with a clear charter
+- Catalyst for change and improving “secure by default”
+- Trusted voice to consumers, producers, and market
+
+> Approaches to achieve the above statement:
+> - (proposed) Update OpenSSF website to reflect updated positioning
+> - (proposed) Ensure we have specific segmentation strategies for targeted audiences
+> - (proposed) Regular TAC / GB interlock with other OSS foundations with regular feedback into (annual?) planning cycles
+> - (proposed) Content / marketing plan with measurements on reach and impact
+> - Alpha/Omega: how to position?
+> - Securing Software Repositories WG
+> - Operator or financial sponsor of common centralized services (e.g. OSS-SIRT, MFA help desk, OpenSSF office hours)
+
+### OpenSSF is an influential advocate for efforts that are aligned with its mission
+- Support development and use of meaningful, actionable standards (e.g. SBOM formats, MFA?)
+- Catalyze improvements / action through targeted investment
+- ...
+
+> Approaches to achieve the above statement:
+> - Oversees time-bound projects to advocate for specific outcomes, e.g.
+>     - Great MFA distribution project
+>     - Sponsor of audits (via OSTIF & others) for critical projects
+> - TAC endorsement of specific standards-track efforts, to be incorporated across other OpenSSF foundation workstreams
+>     - Direct funding / staffing on tooling and adoption efforts (e.g. SPDX-Python)?
+>     - _question_: would it make sense to mandate use of the standard within foundation-hosted projects?
+> - (proposed) Engagement across platforms where developers already are (meetups, conferences, StackOverflow, etc)
+>     - _idea_: should OpenSSF hire dev-rel resources to staff this?
+
+### Regardless of (programming language, VCS, CI platform, etc), consumers of OSS can rely on clear & consistent trusted signals of:
```

Comment:
```suggestion
### Regardless of (programming language, VCS, CI platform, etc), consumers of OSS can leverage on clear & consistent trusted signals of:
```

Rely implies the work is done by OSS itself, which is not accurate and not representative of the desires of the community, rather that OSS consumers have significant responsibility for consumption of OSS, of which the OpenSSF can assist in defining and integrating those "trusted signals" with projects, **for** consumers.

Reply:
The intention behind my use of the word 'rely' was to describe the existence of the signals and their subsequent use in decision making. I did not mean to convey who would be responsible for doing the work to generate those signals. 

I'm not opposed to switching this to "leverage" (as to me it conveys a very similar intended outcome) but I wonder if your point around responsibility is better described in a value statement (that would span the entirety of this vision), similar to @torgo's suggestion below?

Reply:
I think a value statement will definitely assist in ensuring the intentions in the vision are relayed accordingly.

Reply:
### Consumers of OSS can leverage on clear & consistent trusted signals and understand the security profile of open source content: